cfme/utils/smem_memory_monitor.py

```diff
@@ -1106,6 +1108,11 @@ def graph_appliance_measurements(graphs_path, ver, appliance_results, use_slab,
 
 
 def graph_all_miq_workers(graph_file_path, process_results, provider_names):
+    import matplotlib.dates as mdates
+    import matplotlib.pyplot as plt
+    import matplotlib as mpl
+    mpl.use('Agg')
```

Comment:
According to https://matplotlib.org/faq/usage_faq.html you should call .use before importing pyplot, otherwise it has no effect